entries/a/apnic.net.json

```diff
@@ -0,0 +1,12 @@
+{
+  "APNIC": {
+    "domain": "apnic.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.apnic.net/s/article/How-do-I-enable-disable-two-factor-authentication",
+    "keywords": [
+      "utilities"
+    ]

```

Comment:
I suggest adding a `regions` array

Reply:
Thank you for the feedback. What would you suggest as the region value(s), as RIPE covers Europe, which isn't a region value. ARIN also doesn't have one, although you could argue that could have `ca`, `us` and other values.

Reply:
Sorry, ignore me, I was thinking about RIPE NCC, getting confused. I'll add a region now.

Reply:
Actually, it's fine. ARIN doesn't have any listed.